Lib/typing.py

```diff
@@ -189,12 +192,12 @@ def _check_generic(cls, parameters):
     This gives a nice error message in case of count mismatch.
     """
     if not cls.__parameters__:
-        raise TypeError("%s is not a generic class" % repr(cls))
+        raise TypeError(f"{cls} is not a generic class")
```

Comment:
Missed `!r`. It is missed also in many cases below.

Reply:
Thanks! I noticed this, but forgot to fix.